xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -288,6 +337,56 @@ impl XmtpApiSubscription for Subscription {
     }
 }
 
+type EnvelopeStream = Pin<Box<dyn Stream<Item = Result<Envelope, Error>> + Send>>;
```

Comment:
Out of curiosity, why is the `Pin` needed here?

Reply:
Abortable uses some sort of macro to ensure that the inner field is always pinned.